packages/did-provider-ebsi/src/types.ts

```diff
@@ -43,5 +41,19 @@ export interface ICreateIdentifierArgs {
   kms?: string
   alias?: string
   type?: EbsiDidSpecInfo
-  options?: IKeyOpts
+  options?: {
+    methodSpecificId?: string // method specific id for import
```

Comment:
Is this actually used? If so I am assuming for the r1 key and not the k1 key

Reply:
This is existing code, it's an id that gets asserted and appended to `did:ebsi`. It seems to be allowing an external did to be imported. Not attached to any key. Removed.